.devcontainer/Dockerfile

```diff
@@ -0,0 +1,45 @@
+# [Choice] Go version: 1, 1.16, 1.15
+ARG VARIANT=1
```

Comment:
The rest of the repo is pinned to `1.16` so would be best to stay consistent

Reply:
thanks, fixed!